vendor-libs/repository/com/databasepreservation/dbptk-bindings-siarddk/1.0.0/dbptk-bindings-siarddk-1.0.0.pom

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file is what's left of some old dbptk-bindings-siarddk version. Since the most recent seems to be 1.2.0, the `vendor-libs/repository/com/databasepreservation/dbptk-bindings-siarddk/1.0.0` folder can be deleted.

then `vendor-libs/repository/com/databasepreservation/dbptk-bindings-siarddk/maven-metadata-local.xml` should be changed to contain only the latest versions.